en/user_guide/global_page_blocks.rst

```diff
@@ -0,0 +1,33 @@
+Global Pages and Blocks
+=======================
+
+Global Pages and Blocks concept
+-------------------------------
+
+In Open Orchestra, one of the RAD aspects is to simplify creation of pages by using prefunctionnal blocks.
+These blocks must be configured according to their functionnality (see `Block creation`_ and `Block list`_).
+Some blocks could be founded in many pages with the same configuration parameters like header, footer...
+Global pages are containers which allow to stock global configured blocks.
```

Comment:
Are you sure? Globally configured mean that  the configuration is global. Here it's the configured block which is global. No??